### PR TITLE
Pytest fixes

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

**Comment by blueyed** [comment]
I am looking into integrating coverage again, let's wait for this to be rebased on that then to ensure nothing is missed.

File: paste/fixture.py [hunk 1]
```diff
@@ -90,13 +90,14 @@ def reset(self):
             "SMTP connection not quit")
         self.__class__.existing = None
 
+
```
Comment (cdent) [inline paste/fixture.py]: pep8 says two lines between classes

File: setup.cfg [hunk 2]
```diff
@@ -8,3 +8,6 @@ distribute = register sdist bdist_egg upload pudge publish
 
 [bdist_wheel]
 universal=1
+
+[tool:pytest]
+testpaths = tests
```
Comment (cdent) [inline setup.cfg]: Nice, I hoped there was something like this.

File: tests/test_doctests.py [hunk 3]
```diff
@@ -30,29 +33,26 @@
...
 
-def do_doctest_mod(module):
+@pytest.mark.parametrize('module', modules)
+def test_doctest_mods(module):
     module = simple_import(module)
     failure, total = doctest.testmod(
         module, optionflags=options)
     assert not failure, "Failure in %r" % module
 
+
```
Comment (cdent) [inline tests/test_doctests.py]: and two lines between module level methods and other blocks

File: paste/fixture.py (hunk 1 above)
Comment (blueyed) [reply paste/fixture.py]: That's what I've done here - only fixed pep8 issues with surrounding code.
Might be worth to run black once on the entire code base maybe.

File: tests/test_doctests.py (hunk 3 above)
Comment (blueyed) [reply tests/test_doctests.py]: See above.

File: paste/fixture.py (hunk 1 above)
Comment (cdent) [reply paste/fixture.py]: Sorry, I'm only one coffee into my morning and apparently can't read yet :)

I suspect using black would be a _huge_ change, I'm not sure we want to create that much churn, unless Paste gets truly active again.

Comment (blueyed) [reply paste/fixture.py]: Agreed.
Let's postpone this.

**Pull request merged**